plugins/modules/nsxt_transport_node_collections.py

```diff
@@ -192,7 +191,27 @@ def check_for_update(module, manager_url, mgr_username, mgr_password, validate_c
     if existing_tnc is None:
         return False
     if existing_tnc['compute_collection_id'] == transport_node_collection_with_ids['compute_collection_id'] and \
-        existing_tnc['transport_node_profile_id'] != transport_node_collection_with_ids['transport_node_profile_id']:
+            existing_tnc['transport_node_profile_id'] != \
+            transport_node_collection_with_ids['transport_node_profile_id']:
+        return True
+    if existing_tnc.__contains__('description') and not transport_node_collection_with_ids.__contains__('description'):
+        return True
+    if not existing_tnc.__contains__('description') and transport_node_collection_with_ids.__contains__('description'):
+        return True
+    if existing_tnc.__contains__('description') and transport_node_collection_with_ids.__contains__('description') and \
+            existing_tnc['description'] != transport_node_collection_with_ids['description']:
+        return True
+    if existing_tnc.__contains__('transport_node_profile_name') and \
+            transport_node_collection_with_ids.__contains__('transport_node_profile_name') \
+            and existing_tnc['transport_node_profile_name'] != \
+            transport_node_collection_with_ids['transport_node_profile_name']:
+        return True
+    if existing_tnc.__contains__('tags') and not transport_node_collection_with_ids.__contains__('tags'):
```

Comment:
Bug Fixed - 2645685 - "tags" field is not supported by "nsxt_transport_node_collections" module